pom.xml

```diff
@@ -388,7 +387,7 @@
                 <executions>
                     <execution>
                         <goals>
-                            <goal>validate</goal>
+                            <goal>format</goal>
```

Comment:
Ci should be validating not changing code.  That is what you said validate was for.


Sent by Outlook for Android





On Wed, Oct 28, 2015 at 9:14 PM -0700, "Marvin Froeder" <notifications@github.com> wrote:
> @@ -388,7 +387,7 @@
>                  <executions>
>                      <execution>
>                          <goals>
> -                            <goal>validate</goal>
> +                            <goal>format</goal>

I'm totally anti this change.

CI should not be formatting code.  unless CI start to push the formatted code back, which doesn't seem expected either

---
Reply to this email directly or view it on GitHub:
https://github.com/revelc/formatter-maven-plugin/pull/80/files#r43347551